runtime/vm/JFRChunkWriter.hpp

```diff
@@ -300,14 +323,29 @@ class VM_JFRChunkWriter {
 
 			writeJVMInformationEvent();
 
+			writePhysicalMemoryEvent();
```

Comment:
For the actual constant events it looks like the RI only generates them once. So you could look at the chunk count and generate the constant events for the first chunk